pkg/proxy/ipvs/proxier.go

```diff
@@ -1329,13 +1318,6 @@ func (proxier *Proxier) syncProxyRules() {
 				lps = append(lps, lp)
 			}
 
-			// For ports on node IPs, open the actual port and hold it.
-			for _, lp := range lps {
-				if svcInfo.Protocol() != v1.ProtocolSCTP && lp.Protocol == netutils.UDP {
-					conntrack.ClearEntriesForPort(proxier.exec, lp.Port, isIPv6, v1.ProtocolUDP)
-				}
-			}
```

Comment:
are we sure kube-proxy iptables and ipvs flow are identical and we can mimic the conntrack flushing logic?

Reply:
@aojea I might be wrong here, but as per my knowledge conntrack is an independent linux subsystem, and we do enable conntracking for both proxy backends by setting sysctls. 

ref: https://github.com/kubernetes/kubernetes/pull/115744#pullrequestreview-1297801565

Reply:
so, iptables process the Services information and install the iptables rules,  once those are installed, it flushes the stale conntrack entries that may blackhole traffic to these new programmed Services.
iptables loadbalancing works based on NAT, and NAT uses the conntrack table , that makes it more vulnerable to problems with conntrack stale entries.

I have no idea on how IPVS works, per example, this contrack port deletion here is different to the one in iptables mode ... I think that first we need to document the cases where we flush and why, if we flush before the dataplane is programmed another entry can be created and blackhole traffic

Reply:
> are we sure kube-proxy iptables and ipvs flow are identical and we can mimic the conntrack flushing logic?

Well, the ipvs conntrack logic is currently wrong / out-of-date; that's why we're here. But yes, I believe it _should_ be doing the same thing. (We should merge #115299 to make it clearer what that thing actually is though.)

The set of stale conntrack entries that might exist is a function of the semantics of Kubernetes Services, and the semantics of conntrack. It doesn't depend on the backend implementation; if there is an active UDP service on 172.30.0.10:53, and a conntrack entry saying "drop all UDP packets to 172.30.0.10:53", then either that conntrack entry is interfering with service traffic (so we should delete it) or else the proxy backend is routing traffic in some way that completely bypasses that conntrack entry (so we don't _have to_ delete it, but we _can_). There's nothing a backend could be doing where it would actually _want_ that stale conntrack entry to be there.

Some backends might not generate rules that would cause certain kinds of stale conntrack entries to be generated, but that would just mean that the cleanup would always end up deleting 0 entries of that type. Likewise, it's possible that some future backend will expose additional types of stale conntrack entries that need to be cleaned up in certain cases, but these would have to be conntrack entries that never get generated by the current backends, so again we could just remove those entries regardless of backend.